src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -68,14 +62,14 @@ public void execute_validCommand_success() throws Exception {
         assertCommandSuccess(listCommand, ListCommand.MESSAGE_SUCCESS, model);
     }
 
-    @Test
+    /*@Test
```

Comment:
Just delete if it's not used. Why are you deleting this?

Reply:
I will work on it in a later PR when I'm fixing all the tests